### PR TITLE
Add support for cover

### PR DESCRIPTION
**Pull request by maxcanna**
Add support for `cover` domain entities

image

### CODE REVIEW

File: app/i18n/de-DE.po [hunk 1]
```diff
@@ -10,3 +10,15 @@ msgstr "AUS"
 
 msgid "exe"
 msgstr "EXE"
+
+msgid "opening"
+msgstr "ÖFF"
```
Comment (smirko-dev) [inline app/i18n/de-DE.po:15]:
```suggestion
msgstr "AUF"
```

File: app/i18n/de-DE.po [hunk 2]
```diff
@@ -10,3 +10,15 @@ msgstr "AUS"
...
+
+msgid "open"
+msgstr "ÖFF"
```
Comment (smirko-dev) [inline app/i18n/de-DE.po:18]:
```suggestion
msgstr "AUF"
```

**Review by smirko-dev — COMMENTED** [review]
Btw: Thx for fixing my horrible JS style ... I'm actually C++ developer :)

**Comment by maxcanna** [comment]
> Btw: Thx for fixing my horrible JS style ... I'm actually C++ developer :)

no worries 😅  Actually I was worried to have done too much 🤣  If you appreciate that I'd give it a more robust lint as soon as I have time 🙂

**Pull request merged**